cpu/em_quad_level_frequency_transitions.py

```diff
@@ -35,7 +35,7 @@ class freq_transitions(Test):
 
     :avocado: tags=cpu,power,privileged
     """
-    @skipIf(IS_POWER_NV, "This test only supported on Power9  platform")
+    @skipIf(not IS_POWER_NV, "This test is not supported on PowerVM")
```

Comment:
Nitpick: this code runs for each variant, better to stick to adding `not` at the top so that no processing takes place here.

Reply:
Thanks. Should I send an updated patch? Or can this be merged as is?

Reply:
you can send an updated patch.